src/common/stream_response.rs

```diff
@@ -43,6 +46,9 @@ pub struct Alternatives {
 
     #[allow(missing_docs)]
     pub confidence: f64,
+
+    #[allow(missing_docs)]
+    pub languages: Option<Vec<String>>,
```

Comment:
```suggestion

    #[allow(missing_docs)]
    #[serde(default)]
    pub languages: Vec<String>,
```

Let's avoid nesting a vector inside an `Option`. An empty vector communicates the same information as `Option::is_none`.

Reply:
The codebase needs to be updated either way if SDK consumers are creating these objects themselves, and not just deserializing from us. Either way, we need to implement `Default` on the struct.